doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -907,6 +911,12 @@ Libraries for networking
     * Function :c:func:`wifi_credentials_is_empty` to check if the Wi-Fi credentials storage is empty.
     * New parameter ``channel`` to the structure :c:struct:`wifi_credentials_header` to store the channel information of the Wi-Fi network.
 
+* Added the :ref:`lib_wifi_ready` library that manages Wi-Fi readiness for applications by handling supplicant ready and not ready events.
```

Comment:
Just have bullet "Added" after the Libraries for networking heading and the two entries (softap provision and this) under it similarly to the existing softap provisioning entry. Like this:

* Added:

  * :ref:`lib_softap_wifi_provision` library.
  * :ref:`lib_wifi_ready` library